docs/source/install.rst

```diff
@@ -23,11 +23,22 @@ Source
 
 The following instructions assume you'll be using ucx-py on a CUDA enabled system and is in a `Conda environment <https://docs.conda.io/projects/conda/en/latest/>`_.
 
-Note: UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, and ``libnuma`` (numactl on Enterprise Linux).  Please install these with your Linux system's package manager.
+.. note::
+    UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, and ``libnuma`` (numactl on Enterprise Linux).  Please install these with your Linux system's package manager.  Additionally, we provide an example conda environment which installs necessary dependencies for
```

Comment:
Maybe we should add `libibcm` for those who need IB as well?

Reply:
agreed